bin/admin

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+BIN_DIR=${BIN_DIR:-$(cd "${0%/*}"&&pwd)}
+ADDRESS_DIR=${ADDRESS_DIR:-$PWD/addresses}
+ADDRESS_FILE="${ADDRESS_DIR}/addresses-$(seth chain).json"
+
+export ETH_GAS=${ETH_GAS:-"7000000"}
```

Comment:
If not set let's use a default gas price `export ETH_GAS_PRICE=1100000000` (1.1 gwei)

Reply:
OK, but that would have to be changed in all then.

Reply:
done